examples/aws-provider/composite-resources/eks/README.md

```diff
@@ -0,0 +1,64 @@
+# EKS Composition examples
+
+## EKS with Cluster Autoscaler
+
```

Comment:
Can you open an issue to create this with upbound provider and karpenter, add your findings why karpenter cannot be created with contrib provider?
Then mention here that this example is CA because it is based on the contrib provider and link the issue.

Reply:
Done!